llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -7360,20 +7360,25 @@ SDValue RISCVTargetLowering::LowerOperation(SDValue Op,
     uint64_t Val = Op.getConstantOperandVal(0);
     if (isPowerOf2_64(Val)) {
       uint64_t Log2 = Log2_64(Val);
-      if (Log2 < 3)
+      if (Log2 < 3) {
+        SDNodeFlags Flags;
+        Flags.setExact(true);
```

Comment:
I forgot to pass the Flags to getNode. New patch coming as soon as tests finish.